opm/simulators/flow/EclWriter.hpp

```diff
@@ -649,26 +680,10 @@ class EclWriter : public EclGenericWriter<GetPropType<TypeTag, Properties::Grid>
 
     void endRestart()
     {
-        // We need these objects to satisfy the interface requirements of
-        // member function calc_inplace(), but the objects are otherwise
-        // unused and intentionally so.
-        auto miscSummaryData = std::map<std::string, double>{};
-        auto regionData = std::map<std::string, std::vector<double>>{};
-
-        // Note: calc_inplace() *also* assigns the output module's
-        // "initialInplace_" data member.  This is, semantically speaking,
-        // very wrong, as the run's intial volumes then correspond to the
-        // volumes at the restart time instead of the start of the base run.
-        // Nevertheless, this is how Flow has "always" done it.
-        //
-        // See GenericOutputBlackoilModule::accumulateRegionSums() for
-        // additional comments.
-        auto inplace = this->outputModule_
-            ->calc_inplace(miscSummaryData, regionData,
-                           this->simulator_.gridView().comm());
+        this->outputModule_->calc_initial_inplace(this->simulator_.gridView().comm());
```

Comment:
Maybe add a comment here to remind the reader that this call does nothing if the initial in-place volumes have already been calculated&ndash;i.e., from the T=0 values.

Reply:
comment added.